src/pyramid_jwt/__init__.py

```diff
@@ -32,22 +32,23 @@ def create_jwt_authentication_policy(config, private_key=None, public_key=None,
             algorithm=algorithm,
             leeway=leeway,
             expiration=expiration,
+            audience=audience,
             http_header=http_header,
             auth_type=auth_type,
             callback=callback,
             json_encoder=json_encoder)
 
 
 def set_jwt_authentication_policy(config, private_key=None, public_key=None,
-        algorithm=None, expiration=None, leeway=None,
+        algorithm=None, expiration=None, leeway=None, audience=None,
         http_header=None, auth_type=None, callback=None, json_encoder=None):
```

Comment:
Same as earlier:I am a bit worried about adding a parameter in the middle here: theoretically you break backwards compatibility of someone does not use named parameters.